test/memory/org/apache/cassandra/db/compaction/CompactionAllocationTest.java

```diff
@@ -764,4 +800,135 @@ public void widePartitionsOverlappingRows3() throws Throwable
     {
         testWidePartitions("widePartitionsOverlappingRows3", 3, maybeInflate(24), true, true);
     }
+
+
+    private static void testIndexingWidePartitions(String name,
+                                                   int numSSTable,
+                                                   int sstablePartitions,
+                                                   IndexDef...indexes) throws Throwable
+    {
+        String ksname = "ks_" + name.toLowerCase();
+        SchemaLoader.createKeyspace(ksname, KeyspaceParams.simple(1),
+                CreateTableStatement.parse("CREATE TABLE tbl (k text, c text, v1 text, v2 text, v3 text, v4 text, PRIMARY KEY (k, c))", ksname).build());
+
+        ColumnFamilyStore cfs = Schema.instance.getColumnFamilyStoreInstance(Schema.instance.getTableMetadata(ksname, "tbl").id);
+        Assert.assertNotNull(cfs);
+        cfs.disableAutoCompaction();
+        int rowWidth = 100;
+        int rowsPerPartition = 1000;
+
+        measure(new Workload()
+        {
+            @SuppressWarnings("UnstableApiUsage")
+            public void setup()
+            {
+                cfs.disableAutoCompaction();
```

Comment:
is it necessary to disable automatic compaction twice? it has been already done on line 816.

Reply:
CC @bdeggleston

I think this was a copy/paste, so I'm wondering if it was necessary to disable twice in the original tests like `testTinyPartitions()` et al